test/e2e/framework/ingress/ingress_utils.go

```diff
@@ -873,6 +881,24 @@ func (cont *NginxIngressController) Init() {
 	framework.Logf("ingress controller running in pod %v", cont.pod.Name)
 }
 
+func (cont *NginxIngressController) TearDown() {
+	if cont.lbSvc == nil {
```

Comment:
Instead of all this log, please use: WaitForServiceDeletedWithFinalizer

https://github.com/kubernetes/kubernetes/blob/1e70457df46e1af45bf94ae50f3b66c0893c4972/test/e2e/framework/service/wait.go#L71

Reply:
Done.